test/commands/applications/get.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import {expect, test} from '@oclif/test'
+import {mixAPIServerURL} from '../../mocks'
+import testData from './applications-test-data'
+
+const {
+  applicationsGetResponse,
+  noApplicationsResponse,
+} = testData
+
+describe('applications:get command', () => {
+  describe('applications:get command with valid options', () => {
+    const applicationId = '1'
+    const endpoint = '/v4/apps'
+
+    test
+      .nock(mixAPIServerURL, (api) =>
+        api
+          .get(endpoint)
+          .query({
+            view: 'AV_FULL',
+            filter: applicationId,
+          })
+          .reply(200, applicationsGetResponse)
+      )
+      .stdout()
+      .command(['applications:get', '-M', applicationId])
+    .it('applications:get provides human-readable output for given applicationId', (ctx) => {
```

Comment:
We will need a unit test that deals correctly with a response that contains more than one application and picks up the one that exactly matches the app ID requested by the user.

Reply:
We also need to check that another application that has a display name that contains the desired ID is NOT returned.